README.md

```diff
@@ -7,98 +7,129 @@ OpenStack, etc.
 It provides operations for creating, attaching, mounting, unmounting, detaching and deleting 
 volumes. It also includes the required filesystem operations for this.
 
-To build the project, run:
+## Checkout Brooklyn
 
-    mvn clean install
+This project builds on the open-source project Apache Brooklyn (see
+http://brooklyn.apache.org and https://github.com/apache/brooklyn).
 
 
-## Checkout Brooklyn
+## To build the project, run:
 
-This project builds on the open-source project Apache Brooklyn (see 
-http://brooklyn.apache.org and https://github.com/apache/brooklyn).
+    git clone git@github.com:cloudsoft/brooklyn-blockstore.git
+    mvn clean install
 
 
-## Example
+## Example Usage with Apache Brooklyn
 
-First add the required jars to your Apache Brooklyn release (see "Future Work" for discussion 
-of OSGi): 
+Assuming you build project and you know want to use it in Apache Brooklyn blueprints.
 
-    BROOKLYN_HOME=~/repos/apache/brooklyn/brooklyn-dist/dist/target/brooklyn-dist/brooklyn/
     BROOKLYN_BLOCKSTORE_REPO=~/repos/cloudsoft/brooklyn-blockstore
-    BROOKLYN_BLOCKSTORE_VERSION=0.6.0-SNAPSHOT
-    
-    cp ${BROOKLYN_BLOCKSTORE_REPO}/blockstore/target/brooklyn-blockstore-${BROOKLYN_BLOCKSTORE_VERSION}.jar ${BROOKLYN_HOME}/lib/dropins/
-
-And launch Brooklyn:
-
-    ${BROOKLYN_HOME}/bin/brooklyn launch
-
-Then deploy an app. The example below creates a VM with a new volume:
-
-    location: aws-ec2:us-east-1
-    services:
-    - type: org.apache.brooklyn.entity.software.base.VanillaSoftwareProcess
-      brooklyn.config:
-        launch.command: |
-          sudo chown -R myname /mount/brooklyn/h/
-          echo abc > /mount/brooklyn/h/myfile.txt
-        checkRunning.command: echo checked
-        provisioning.properties:
-          user: myname
-          customizers:
-          - $brooklyn:object:
-              type: brooklyn.location.blockstore.ec2.Ec2NewVolumeCustomizer
-              object.fields:
-                volumes:
-                - blockDevice:
-                    sizeInGb: 1
-                    deviceSuffix: 'h'
-                    deleteOnTermination: false
-                    tags:
-                      brooklyn: br-example-aled-1
-                  filesystem:
-                    mountPoint: /mount/brooklyn/h
-                    filesystemType: ext4
+    cd ${BROOKLYN_BLOCKSTORE_REPO}
+
+    echo Follow Apache Brooklyn Getting Started instructions from https://brooklyn.apache.org/#get-started
+
+Create a catalog item and put location configuration for extending hard drives.
+The example below creates a VM with a new volume:
+
+    brooklyn.catalog:
+      brooklyn.catalog:
+        id: my-app.io.cloudsoft.brooklyn-blockstore.new-volume-customizer
+        version: 0.6.0-SNAPSHOT
+        brooklyn.libraries:
+        - file://home/myUser/repos/cloudsoft/brooklyn-blockstore/target/brooklyn-blockstore-0.6.0-SNAPSHOT.jar
```

Comment:
If the instructions recommend building locally, better use `~/.m2/repository/....`.